### ISSUE TITLE
system-tests: create test for REST API - address resource required params are missing

### ISSUE THREAD

**Issue by kornys** [EnMasseProject/enmasse#1034]
create tests for correct response when required parameters are not set (new issue)

"spec": {
        "address": "my_queue",  // Required. Default is .metadata.name
        "type": "queue",        // Required. Valid types are defined by the AddressSpace
        "plan": "medium"        // Required. Valid plans can be found using /apis/enmasse.io/v1/schema
    }

tracker https://github.com/EnMasseProject/enmasse/issues/936

**Comment by tkratky** [comment]
blocked by: https://github.com/EnMasseProject/enmasse/issues/1082

**Comment by tkratky** [comment]
done as part of: https://github.com/EnMasseProject/enmasse/pull/1101
